### PR TITLE
[tc-lib-postgres] Assert that methods names are unique

### PR DESCRIPTION
**Pull request by helfi92**

### CODE REVIEW

File: libraries/postgres/test/database_test.js [hunk 1]
```diff
@@ -101,6 +101,45 @@ dbSuite(path.basename(__filename), function() {
...
+          },
+          testdata: {
+            mode: 'read',
+            serviceName: 'service-2',
+            args: '',
+            returns: 'void',
+            body: `begin
+              insert into testing values (1, 2), (3, 4);
+            end`,
+          },
```
Comment (djmitche) [inline libraries/postgres/test/database_test.js]: I'm curious how this test case works: doesn't that create a JS object with just one property?

Comment (helfi92) [reply libraries/postgres/test/database_test.js]: oo, you're right. I actually haven't tested this. I was relying on CI to run the tests for me. It's probably a good time for me to read the README and set up the database.

**Comment by helfi92** [comment]
@djmitche It turns out that `safeLoad` disallows duplicate key fields which we use in the `fromDbDirectory` method.

**Comment by djmitche** [comment]
Awesome :)

**Pull request merged**